media/css/pocket/components/about.scss

```diff
@@ -35,8 +35,12 @@ $pocket-image-path: '/media/img/pocket';
         z-index: 2;
 
         .about-section-heading {
+            @include font-size(28px);
             font-family: $font-sans-alt;
             font-weight: 500;
+
+            // need to increase line-height here to remove visual bug on non EN locales (mainly an issues with the letter 'j'
+            line-height: 60px;
```

Comment:
Suggestion (non-blocking): it's best practise to try and avoid using `px` as a value for `line-height`, as it becomes possible to set a value that is smaller than the actual font size this way. Instead suggest using a multiplier, such as `line-height: 1.3;`